src/Elastic.OpenTelemetry/Elastic.OpenTelemetry.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What is the actual change here? Is this CRLF vs LF line endings? Just curious.

Reply:
I'm not sure. Our formatting check didn't find any errors, but perhaps some whitespace got changed.

Reply:
Ah. Your change removed the `\357\273\277`  UTF-8 BOM that the file had had.  My understanding is that UTF-8 files generally do NOT need a BOM.  Unicode doc says "Use of a BOM is neither required nor recommended for UTF-8".

/shrug

Reply:
Ah, that makes sense. We try to enforce that. I'm not sure what triggered it here.